### PR TITLE
Throw `PlatformNotSupportedException` in `netstandard1.3`

### PR DESCRIPTION
**Pull request by dougbu**
- stop trying to use DCS in the JSON and XML formatters
- if `UseDataContractSerializer` or `!UseXmlSerializer`:
  - `CanReadType(...)` and `CanWriteType(...)` always return `false`
  - `ReadFromStreamAsync(...)` and `WriteToStreamAsync(...)` always `throw`
- change default XML formatter configuration to use `XmlSerializer`
- adjust and disable tests in reaction
- add tests of new `netstandard1.3` behavior
- nits:
  - correct namespace of `SerializerConsistencyTests`; was alone in previous namespace
  - s/DataContractFormatter/XmlSerializerFormatter/ to reflect actual `XmlMediaTypeFormatter` test actions

### CODE REVIEW

**Comment by dougbu** [comment]
Thanks @mconnew. Unfortunately, it appears only a select few have write access in this repo 😦 That means I need another reviewer.

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**
